Makefile

```diff
@@ -30,7 +30,8 @@ parser:
 	a=`mktemp temp.XXXXXX`; \
 	goyacc -o /dev/null -xegen $$a parser/parser.y; \
 	goyacc -o parser/parser.go -xe $$a parser/parser.y; \
-	rm -f $$a; \
+	rm -f $$a;
+	@grep " conflict on " y.output | awk '{print} END {if (NR > 0) {print "Find " NR " conflict in parser.y. Please check y.output for more information."; exit 1}}'
```

Comment:
checking the goyacc output like `2>&1 | grep "conflicts"` for above command may be more fast than grep whole output file.